librb/rbRadios.py

```diff
@@ -180,4 +177,3 @@ def station_search(self, params, **kwargs):
         kwargs["params"] = params
         endpoint = self.builder.produce_endpoint(endpoint="station_search")
         return request(endpoint, **kwargs)
-        
```

Comment:
newline at the end is PEP8

Reply:
This is some git trickery since I have a newline even after git reset --hard :S